caas/kubernetes/clientconfig/k8s_test.go

```diff
@@ -219,14 +232,23 @@ func (s *k8sConfigSuite) TestConfigErrors(c *gc.C) {
 			errMatch: `failed to read credentials from kubernetes config: AuthInfo: "the-user" with both Token and User/Pass not valid`,
 		},
 		{
-			title: "emptyClientKeyDataInvalidConfig",
+			title: "emptyTokenInvalidConfig",
+			configYamlContent: `
+- name: the-user
+  user:
+    client-certificate-data: QQ==
+    client-key-data: CK==
+`,
+			errMatch: `failed to read credentials from kubernetes config: no Token found for "the-user" with auth type "oauth2withcert" not valid`,
```

Comment:
the error message is poorly worded because of the not valid error construct mentioned above

Reply:
updated